ui-admin/src/study/participants/enrolleeView/EnrolleeTimeline.tsx

```diff
@@ -96,3 +118,18 @@ export default function EnrolleeTimeline({ enrollee, studyEnvContext }:
   </div>
 }
 
+/**
+ * Renders an icon to indicate whether an email has been opened
+ */
+export function renderEmailActivityIcon(row: Notification) {
+  const notificationEventDetails = row.eventDetails
+  if (notificationEventDetails && notificationEventDetails.opensCount > 0) {
+    return <FontAwesomeIcon icon={faEnvelopeOpen} aria-label={'Email opened'}/>
+  }
+
+  if (notificationEventDetails) {
+    return <FontAwesomeIcon icon={faEnvelope} aria-label={'Email not yet opened'}/>
+  }
+
+  return <>N/A</>
```

Comment:
for a softer n/a look
```suggestion
  return <span className="fw-light fst-italic">n/a</span>
```